controllers/src/main/scala/cromwell/pipeline/controller/utils/auth/SecurityDirective.scala

```diff
@@ -1,9 +1,10 @@
-package cromwell.pipeline.datastorage.utils.auth
+package cromwell.pipeline.controller.utils.auth
```

Comment:
I'm not sure it should be in `utils`. Probably just `auth` will be enough.

Reply:
moved